tests/console/just_install_packages.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
```

Comment:
I would not know from the name of this test module that is related with libyui, could you think in other options, i.e.: install_libyui_packages.

Reply:
The module can be used to install any package. The pre-existing modules are doing some extra checks , therefore I named it just_install_packages, so that it's clear for people who already use old modules. But, I'm open to name proposals.

Reply:
oh I see the mess of existing names, thanks for pointing me to that. For me would just work to use 'verb' first, so could be for example: 'install_packages_only', 'install_package_list' or 'install_whatever_you_pick'. It is like for naming method or variable, to avoid conjuntions, etc. they are not common.